Employment Policies/Working Remotely.md

```diff
@@ -39,13 +39,13 @@ The majority of policies that support remote work in Clef apply to everyone at C
 
 #### Extended remote work 
 
-***An extended remote work period includes anything longer than 2 days or any period of time where we work from a irregular place.***
+***An extended remote work period includes anything longer than 2 days or any period of time where we work from somewhere place.***
 
-If you're planning to work remotely for more than two consecutive days, or you're planning to work remotely from a place that's abnormal (i.e. from a partner's home in a different city), you'll need to follow these additional guidelines. Even more than standard remote work, working remotely for an extended period of time is a privilege — it **will** require a large amount of extra work from you to make sure that the team is effective with you working remotely.
+If you're planning to work remotely for more than two consecutive days, or you're planning to work remotely from a place that's not your home (i.e. from a partner's home in a different city), you'll need to follow these additional guidelines. Even more than standard remote work, working remotely for an extended period of time is a privilege — it **will** require a large amount of extra work from you to make sure that the team is effective with you working remotely.
 
 Get Approval From your manager
 
-You need your manager's approval before you can work remotely for an extended period of time. This check-in should happen at least two weeks in advance, before you make any travel arrangements, and you should come prepared with a plan for how you will handle all the edge cases involved in extended remote work. In order for an employee to work remotely for an extended period of time, they should have demonstrated in the past that they are effective working remotely and upholding their quality of work. If that's not the case, the manager can and should veto the option and work with the employee to come up with a plan that would get them on track to work remotely in a more effective way.
+You need your manager's approval before you can work remotely for an extended period of time. This check-in should happen at least two weeks in advance, before you make any travel arrangements. In order for an employee to work remotely for an extended period of time, they should have demonstrated in the past that they are effective working remotely and upholding their quality of work. If that's not the case, the manager can and should veto the option and work with the employee to come up with a plan that would get them on track to work remotely in a more effective way.
```

Comment:
oops, my comment was unclear, I meant that the plan to get them back on track didn't make sense, not the plan to handle edge cases